src/dataflow/src/render/delta_join.rs

```diff
@@ -20,7 +21,7 @@ use repr::{Datum, Row, RowArena};
 use super::context::{ArrangementFlavor, Context};
 use crate::operator::CollectionExt;
 
-impl<G> Context<G, RelationExpr, Row, Timestamp>
+impl<'g, G> Context<Child<'g, G, G::Timestamp>, RelationExpr, Row, Timestamp>
```

Comment:
What happened here?

Reply:
Some things need to be impl'd on

```
Context<G, RelationExpr, Row, Timestamp>
```

while others need to be on:

```
Context<Child<'g, G, G::Timestamp>, RelationExpr, Row, Timestamp>
```

I somewhat inadvertently moved the impl of `ensure_rendered` into the latter, which forced the impl of `render_delta_join` to also move to the latter, since it calls `ensure_rendered`. Fixed by moving the impl of `ensure_rendered` back to the former, which makes everything work out a bit more cleanly.